From 26f7f90fa477dbd4de32d17bce38c6fe6306134c Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 25 Oct 2017 15:48:03 +0800
Subject: [PATCH] ARM: rockchip_defconfig: disable DEVKMEM

42019da52a83 ("android: base-cfg: turn off /dev/mem and /dev/kmem")

Change-Id: I2d34d76b6ed85fbd590be2e0c445c19f253bc2c4
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 894de33d939b..f6e60e4d9b95 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -342,6 +342,7 @@ CONFIG_INPUT_GPIO=y
 # CONFIG_SERIO is not set
 # CONFIG_VT is not set
 # CONFIG_LEGACY_PTYS is not set
+# CONFIG_DEVKMEM is not set
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
 CONFIG_SERIAL_8250_CONSOLE=y
-- 
2.35.3

